cpp/board_class_boggler.h

```diff
@@ -34,6 +34,7 @@ class BoardClassBoggler {
   Trie* dict_;
   char bd_[M * N][27];  // null-terminated lists of possible letters
   unsigned int used_;
+  unsigned int used_ordered_; // used cells mapped to their split order
```

Comment:
This is irrelevant to `BucketBoggler` (`ibuckets.h`), the other subclass, so it should go in `OrderlyTreeBuilder`.